docker/ex2/Dockerfile

```diff
@@ -0,0 +1,10 @@
+FROM ruby:2.7
+
+WORKDIR /app
+
+RUN gem install json faker
+
+COPY ex2.rb /app/
+
+CMD ["ruby", "ex2.rb" , "arg1","arg2"]
```

Comment:
These args are not used in the code,  pass some valid data and use it in the code